src/index.css

```diff
@@ -0,0 +1,12 @@
+html {
```

Comment:
It's a good idea to have generic styles for tags. I suggest you have a look at the inverted triangle css architecture.